packages/rancher-logging/app-readme.md

```diff
@@ -0,0 +1,3 @@
+Rancher Logging, powered by the [Bonzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/), allows users to configure complex logging pipelines with a few simple resources.  To collect logs from a single namespace, users can create an [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/), to define where logs are stored, and 
+corresponding [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/), to describe what logs to include and what outputs to send to.  If you want to collect logs from 
+the entire cluster, you can similarly define a [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) and [ClusterOutput](ttps://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/). 
```

Comment:
Couple of small comments, but lgtm overall:
1. The second sentence and third sentence should read the same (i.e. talk about ClusterOutput then ClusterFlow, just like you do Output then Flow, or vice versa.
1. Second sentence has unnecessary commas.
1. Not sure why the lines are split up in the middle of sentences.

```suggestion
Rancher Logging, powered by the [Bonzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/), allows users to configure complex logging pipelines with a few simple resources.
To collect logs from a single namespace, users can create an [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) to define where logs are stored and a corresponding [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) to describe what logs to include and what outputs to send to.
To collect logs from the entire cluster, users can similarly create a [ClusterOutput](ttps://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) and a corresponding [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/). 
```